Extend the attribs field to mediumtext/text in MySql and Postgres

pixxelfactory opened the pull request:
Pull Request for Issue # .

### Summary of Changes
When using a content-plugin to add fields to the core content of joomla, if the user adds more content than 5160 characters, the attribs json will be truncated.
Since in the most cases using a plugin is necessary (the Field Groups / Fields ecc. do not have all functionality wich is needed on many sites), i extended the two db-fields to mediumtext / text.

pixxelfactory committed:
Extend attributes field to mediumtext

When using a content-plugin to extend the standard joomla-content functionality, after 5000 characters the contents are truncated. Extend the attribs field to mediumtext

pixxelfactory committed:
Extend attribs field to text

To prevent truncate problems when using a content-plugin to extend core joomla-functionality.

SharkyKZ commented:
IMO, `text` is fine (and consistent with `params` columns in other tables) but `mediumtext` sounds overkill. If you're storing that much JSON data, you're probably better off creating a separate table for your plugin.

pixxelfactory committed:
Change attribs from mediumtext to text

As mentioned in the Pull-Request thread, mediumtext is a bit of an overkill, so change to text

pixxelfactory commented:
Changed mediumtext to text

alikon commented:
you need to add the update sql script too

pixxelfactory commented:
You mean an update-script wich changes the existing varchar field to text (if a user updates joomla?)
Where do i need to put that?
Thanks and best regards

SharkyKZ commented:
Here https://github.com/joomla/joomla-cms/tree/7264165c4ae219a863d7684ce8960fa062e01f50/administrator/components/com_admin/sql/updates

pixxelfactory committed:
Update attribs field to text

pixxelfactory committed:
Increase attribs column to text

pixxelfactory commented:
I've added the two sql-scripts, i'm not sure if the Postgres update-script is 100% correct, since i usually do not use Postgres.
Another question: Why is the continous-integration/drone/pr - Build failing?

HLeithner commented:
Can you please rebase this for j4, thx

franz-wohlkoenig commented:
closed for rebase on j4.

brianteeman commented:
@HLeithner It would be really helpful if you could explain to the person who took the time to submit a pull request why you have closed this and asked for it to be rebased for Joomla 4.
